src/main/java/by/itechart/api/controller/UserController.java

```diff
@@ -1,32 +1,49 @@
 package by.itechart.api.controller;
 
-import by.itechart.api.entity.User;
+import by.itechart.api.dto.UserDTO;
+import by.itechart.api.service.impl.UserServiceImpl;
+import lombok.RequiredArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
-
+import org.springframework.web.bind.annotation.*;
 import java.util.List;
 
 @RestController
+@RequiredArgsConstructor
 @RequestMapping("/users")
 public class UserController {
 
+    private final UserServiceImpl userService;
+
     @GetMapping("/all")
-    public ResponseEntity<List<User>> getAllUsers() {
-        //TODO return users, not just status
-        return new ResponseEntity<>(HttpStatus.OK);
+    public ResponseEntity<List<UserDTO>> getAllUsers() {
+        return new ResponseEntity<>(userService.findAll(), HttpStatus.OK);
     }
 
     /**
      * status FOUND here is just for test. It will be changed later
      * this method will get a authentication parameter later, but for now there will be no params
+     *
      * @return current user
      */
     @GetMapping("/current")
-    public ResponseEntity<User> getCurrentUser() {
-        //TODO return current user, not just status
+    public ResponseEntity<UserDTO> getCurrentUser() {
         return new ResponseEntity<>(HttpStatus.FOUND);
     }
+
+    @PatchMapping("{id}")
```

Comment:
Does request mapping work without leading slash?

Reply:
> Does request mapping work without leading slash?

Yeah, it works!